src/schematools/types.py

```diff
@@ -99,38 +112,38 @@ def url_prefix(self) -> str:
         return self.get("url_prefix", "")
 
     @property
-    def identifier(self):
+    def identifier(self) -> str:
         """Which fields acts as identifier. (default is Django "pk" field)"""
         return self.get("identifier", "pk")
 
     @property
-    def version(self):
+    def version(self) -> str:
```

Comment:
```suggestion
    def version(self) -> Optional[str]:
```

Reply:
There's more to it. Let me look something up

Reply:
No, `version` is a required property according to the meta schema. Hence we should always be able to return a version. Meaning `-> str` is the correct return type

Reply:
No, version is a required property according to the meta schema. Hence we should always be able to return a version. Meaning -> str is the correct return type